app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -783,7 +783,7 @@ private CharSequence getPollDescription(Context context,
         if (poll == null) {
             return "";
         } else {
-            CharSequence[] args = new CharSequence[5];
+            Object[] args = new CharSequence[5];
```

Comment:
Huh?

Reply:
Makes the warning go away. But not pretty at all, do you have a better idea?